src/apps/vpn/appsettingslist.h

```diff
@@ -54,7 +58,8 @@ SETTING_STRINGLIST(captivePortalIpv4Addresses,        // getter
                    "captivePortal/ipv4Addresses",     // key
                    QStringList(),                     // default value
                    false,                             // user setting
-                   false                              // remove when reset
+                   false,                             // remove when reset
+                   true                               // sensitive (do not log)
```

Comment:
this is the mozilla IP address. for: detectportal.firefox.com. There is nothing sensitive here..